config.example.cfg

```diff
@@ -1,4 +1,4 @@
 DEBUG = True
```

Comment:
This change should not be done in the example config file. Please remove it from your pull request.